python/pyarrow/tests/test_array.py

```diff
@@ -34,6 +34,9 @@
 from pyarrow.vendored.version import Version
 
 
+@pytest.mark.skipif(
+    sys.platform == "emscripten", reason="Emscripten can't run processes"
+)
```

Comment:
You added a `processes` marker I saw in contest.py. Could/should that be used here instead?